### ISSUE TITLE
Is it possible to set the parameterConstraints as a metadata file

### ISSUE THREAD

**Issue by andresionek91** [microcks/microcks#1371]
### Reason/Context

Is it possible to add header/parameter constraints via the metadata file like below?

```
---
apiVersion: mocks.microcks.io/v1alpha1
kind: APIMetadata
metadata:
  name: pet
  version: 1.0.0
  labels:
    domain: pets

operations:
  GET /pet:
    delay: 0
    dispatcher: FALLBACK
    dispatcherRules: |-
      {
        "dispatcher": "URI_PARAMS",
        "dispatcherRules": "petId",
        "fallback": "NOT FOUND"
      }
    parameterConstraints:
      - name: Authorization
        in: header
        required: true
        recopy: false
        mustMatchRegexp: "^Bearer\\s[a-zA-Z0-9\\._-]+$"

```

### Description

I'm trying to set the constraints via a metadata file import, but it doesn't work. Is it possible to do it? If so, what's the syntax?

**Comment by lbroudoux** [comment]
Hello! It's not possible as of today but it has been discussed on Discord and it's also tracked here: #1366

**Comment by lbroudoux** [comment]
Hey @andresionek91 

I started working on this and I wanted your feedback on one behavior I implemented. As many artifacts could/would contribute to constraints, we have to handle collisions or duplicates. 

As of today, I considered the constraint `name` and `in` couple to represent identity. Typically this mean that if you have a first artifact specifying something like:
```yaml
- name: Authorization
   in: header
   required: true
   recopy: false
   mustMatchRegexp: "^Bearer\\s[a-zA-Z0-9\\._-]+$"
```
and another specifying:
- name: Authorization
   in: header
   required: true
   recopy: true
   mustMatchRegexp: "^Basic\\s[a-zA-Z0-9\\._-]+$"
```
The last imported artifact will always overwrite the value of the previous one as both constraints relate to `Authorization in header`.

Does is sound good to you?
